### PR TITLE
Use article headline rather than front headline in emails

### PR DESCRIPTION
**Pull request by philmcmahon**
## What does this change?
Modifies the `?format=email-headline` endpoint so that it returns the webTitle of the first curated piece of content in a front - previously it was using the 'fronts header headline' - i.e. in [this](https://m.code.dev-theguardian.com/email/uk/daily) piece it would be 'bolt comfortably wins....' as opposed to 'Usain Bolt comfortably...' which is the [full headline](https://m.code.dev-theguardian.com/sport/2015/may/26/usain-bolt-200m-golden-spike-ostrava)

Theresa Malone requested this change.

This change modifies the endpoint so that if it fails to find a title, it returns a not found response rather than a 200 with an error message - see [here](https://github.com/guardian/frontend/pull/19793/commits/2b52676840c964f9e533723ca67c1dff17a7e1c5).

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by shtukas — APPROVED** [review]

**Comment by QuarpT** [comment]
We should look in backfill if curated is empty

**Comment by philmcmahon** [comment]
Good point @QuarpT - have updated

File: facia/app/controllers/FaciaController.scala [hunk 1]
```diff
@@ -123,11 +123,11 @@ trait FaciaController extends BaseController with Logging with ImplicitControlle
...
             if (shouldOnlyReturnHeadline(request)) {
-              val headline = for {
+              val webTitle = for {
                 topCollection <- faciaPage.collections.headOption
-                topCurated <- topCollection.curated.headOption
-              } yield topCurated.header.headline
-                RevalidatableResult.Ok(headline.getOrElse("Error: Could not extract headline from front"))
+                topCurated <- topCollection.curatedPlusBackfillDeduplicated.headOption
+              } yield topCurated.properties.webTitle
+              webTitle.map(RevalidatableResult.Ok(_)).getOrElse(WithoutRevalidationResult(NotFound("Could not extract headline from front")))
```
Comment (QuarpT) [inline facia/app/controllers/FaciaController.scala]: I'd put the `RevalidatableResult.Ok(_)` inside the yield 😆

**Review by QuarpT — APPROVED** [review]

**Comment by shtukas** [comment]
+1

**Pull request merged**